baselines/fedper/fedper/conf/cifar10/resnet/fedavg.yaml

```diff
@@ -0,0 +1,44 @@
+---
+num_clients: 10 # total number of clients
+num_epochs: 4 # number of local epochs
+batch_size: 128
+num_rounds: 50
+clients_per_round: 10
+learning_rate: 0.01
+algorithm: fedavg
+model_name: resnet
+
+client_resources:
+  num_cpus: 4
+  num_gpus: 1
+
+server_device: cuda:0
+
+dataset:
+  name : "cifar10"
+  split: sample
+  num_classes: 10
+  seed: 42
+  num_clients: ${num_clients}
+  fraction: 0.83
+
+model:
+  _target_: fedPer.implemented_models.mobile_model.MobileNet
```

Comment:
is this correct? shouldn't it point to the resnet? (this applies to 4 config files)